src/Microsoft.VisualStudio.Editors/PropPages/ReferencePropPage.vb

```diff
@@ -781,6 +785,9 @@ Namespace Microsoft.VisualStudio.Editors.PropertyPages
         End Function
 
         Private Function GetCurrentImportsList() As String()
+            'TODO : Remove the below empty return after GetCurrentImportsList() is implemented 
+            Return New String() {}
+
             Dim theVSProject As VSLangProj.VSProject
             Dim _Imports As VSLangProj.Imports
```

Comment:
This will impact regular projects because it's the same property page. I think it's best to not comment these out and leave the page crashing for us to make sure that we don't inadvertently ship something broken for existing projects.